### DISCUSSION TITLE
Moving changes between files

### DISCUSSION THREAD

**Discussion by SymmetricDesigns** [moshfeu/vscode-compare-folders#148] [Q&A]
Why can I only copy changes in one direction?
I'm trying to get some changes from a different repository. When I select the other repository and look at the changes I can only copy changes TO that repository not FROM it. Not only does this seem backwards, but I should be able to do either. Am I missing something? I solved it by opening the repo with the changes in my workspace instead and then copying the changes out of it into the other repo. This seem counter-intuitive. I also tried loading both repos into my workspace, but it seemed to always choose the direction I didn't want.

**Comment by moshfeu** [comment]
Hi @SymmetricDesigns 

You are right! I'm usually start the process just by selecting 2 folders so I'm not experiencing the "my folder" vs "compared folder" experience so it feel less weird

For now, you can swap sides

<img width="387" alt="Screenshot 2024-04-02 at 21 16 34" src="https://github.com/moshfeu/vscode-compare-folders/assets/3723951/9c3ee6c1-61ac-4a4c-a98b-cff624ad4760">

Also, if you compare 2 folders from the Explorer (select 2 folders in the explorer panel, right click and `[Compare Folders] Compare selected folders`) it considers the order of the selection

<img width="449" alt="Screenshot 2024-04-02 at 21 18 43" src="https://github.com/moshfeu/vscode-compare-folders/assets/3723951/22636664-435b-4a53-a3de-29a64e885d28">

I might add a setting option to control the sides since I believe that a lot of people are used to it and I don't want to change that for them out of no where

Thanks for letting me know and for taking the time to discuss that matter, really appreciated
